### PR TITLE
SIDM 4211

### PR DESCRIPTION
**Pull request by jitumiah**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SIDM-4211

### Change description ###
added error handling for stale user login

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -126,6 +126,9 @@ public.login.error.locked.title=There is a problem with your account login detai
 public.login.error.locked.instruction=Your account is locked due to too many unsuccessful attempts.<br>You can <a href="{0}">reset your password</a> to unlock your account.
 public.login.error.suspended.title=There is a problem with your account login details
 public.login.error.suspended.instruction=Your account has been blocked. Contact us to get help signing in.
+#TODO update once we get the proper message back from hmcts ux
+public.login.error.retired.title=There is a problem with your account login details
+public.login.error.retired.instruction=Your account has been deactivated. You have been sent a registration email.
```
Comment (dfourn) [inline src/main/resources/messages.properties:131]: I know it's a placeholder message, but I think it should say something more like:
"Your account has been deactivated due to inactivity. Please reset your password using the link sent to your email to reactivate it."

File: src/main/webapp/WEB-INF/jsp/login.jsp [hunk 2]
```diff
@@ -68,6 +68,19 @@
                                 </p>
                             </div>
                         </c:when>
+                        <c:when test="${isAccountRetired}">
+                            <script>
+                                sendEvent('Authorization', 'Error', 'Account is retired, stale user has been sent registration');
```
Comment (dfourn) [inline src/main/webapp/WEB-INF/jsp/login.jsp:73]: do we even use/need these events at all?

Comment (jitumiah) [reply src/main/webapp/WEB-INF/jsp/login.jsp:73]: @dfourn looks like it is for google analytics. Not sure if it is being used by anyone.

File: src/main/resources/messages.properties (hunk 1 above)
Comment (jitumiah) [reply src/main/resources/messages.properties]: @dfourn applied suggested changes

File: src/main/resources/messages_cy.properties [hunk 3]
```diff
@@ -127,6 +127,9 @@ public.login.error.locked.title=Mae problem gyda’ch manylion mewngofnodi
 public.login.error.locked.instruction=Mae eich cyfrif wedi’i gloi oherwydd gormod o ymdrechion aflwyddiannus i ddefnyddio'ch cyfrif.<br>Gallwch  <a href="{0}">ailosod eich cyfrinair</a> i ddatgloi eich cyfrif.
 public.login.error.suspended.title=Mae problem gyda’ch manylion mewngofnodi
 public.login.error.suspended.instruction=Mae eich cyfrif wedi cael ei flocio.  Cysylltwch â ni i gael cymorth i fewngofnodi.
```
Comment (nikola-naydenov-hmcts) [inline src/main/resources/messages_cy.properties:129]: I dont mind this if we have a ticket in the sprint to make sure we fix this before we proceed

Comment (jitumiah) [reply src/main/resources/messages_cy.properties:129]: @nikola-naydenov-hmcts 
Created a ticket in the backlog against user licensing so we don't forget
https://tools.hmcts.net/jira/browse/SIDM-4391

**Pull request merged**